docs/monitoring-aws-lambda.asciidoc

```diff
@@ -88,13 +88,19 @@ The configured name of your application or service.  The APM agent will use this
 === `ELASTIC_APM_DATA_RECEIVER_TIMEOUT_SECONDS`
```

Comment:
Should we add deprecation notices to the docs for the existing config? https://github.com/elastic/docs#additions-and-deprecations

Reply:
Good point! Added 👍